routers/facebook_api.py

```diff
@@ -146,15 +153,31 @@ def fb_connect_redirect(request: Request):
             status_code=400,
         )
 
+    # only connect to pages that are not already connected
```

Comment:
Any reason for doing this?

Reply:
reverting for now as it seems to keep pages that are disconnected from facebook around as integrations

Reply:
Ok yeah dont wanna loose the analytics for those integrations, fixed in https://github.com/GooeyAI/gooey-server/commit/c08a3c6ad00d7445804b456356515a6edf906195